etna/core/fields.py

```diff
@@ -80,6 +80,8 @@ def __init__(
 
         super().__init__(fields=fields, **kwargs)
 
+        self.label_suffix = ""  # Removes : as label suffix
+
```

Comment:
Where did this requirement come from, @TNA-Allan?

Reply:
Have updated the PR ticket. This is come from DF-702 - https://national-archives.atlassian.net/browse/DF-702?focusedCommentId=49550.